public/js/shared.js

```diff
@@ -89,6 +90,42 @@ function showDetails() {
   }
 }
 
+/**
+ * Render the policy value selection bar html string based on policy object
+ * given.
+ *
+ * @param {Object} policy
+ * @return {string}
+ */
+function policyValueSelector(policy) {
+  const desc = policy.usage_desc ? html `<span>${policy.usage_desc}:</span>` : '';
+  const optionButtons = Object.entries(policy.usage)
+      .sort(([ka, va], [kb, vb]) => ka < kb)
+      .map(([policyValue, _]) => {
+        const currentPolicyValue = new URL(location).search.slice(1);
+        // Note: inline 'active' attribute as a template argument
+        // will result in lithtml not correctly render the element.
+        if (policyValue.localeCompare(currentPolicyValue) === 0) {
+          return html `
+          <a href="${policy.url}?${policyValue}"
+            class="try-button" onclick=updatePolicyValue(this) active
```

Comment:
I think you should be able to use [ifDefined](https://lit-html.polymer-project.org/guide/template-reference#ifdefined) to avoid having the repetitive if/else branches.

Something like this, although you may need to experiment with the best "defined" value:

```suggestion
            class="try-button" onclick=updatePolicyValue(this) active="${ifDefined(policyValue.localeCompare(currentPolicyValue) === 0 ? true : undefined)}"
```

Reply:
Done.